docs/usage/types.md

```diff
@@ -564,12 +567,13 @@ _(This script is complete, it should run "as is")_
 
 For URI/URL validation the following types are available:
 
-- `AnyUrl`: any scheme allowed, TLD not required
-- `AnyHttpUrl`: scheme `http` or `https`, TLD not required
-- `HttpUrl`: scheme `http` or `https`, TLD required, max length 2083
-- `PostgresDsn`: scheme `postgres` or `postgresql`, user info required, TLD not required
-- `RedisDsn`: scheme `redis` or `rediss`, user info not required, tld not required (CHANGED: user info
-  not required from **v1.6** onwards), user info may be passed without user part (e.g., `rediss://:pass@localhost`)
+- `AnyUrl`: any scheme allowed, TLD not required, host required
+- `AnyHttpUrl`: scheme `http` or `https`, TLD not required, host required
+- `HttpUrl`: scheme `http` or `https`, TLD required, host required, max length 2083
+- `FileUrl`: scheme `file`, host not required
+- `PostgresDsn`: scheme `postgres` or `postgresql`, user info required, TLD not required, host required
+- `RedisDsn`: scheme `redis` or `rediss`, user info not required, TLD not required, host not required (CHANGED: user
+  info not required from **v1.6** onwards), user info may be passed without user part (e.g., `rediss://:pass@localhost`)
```

Comment:
I reckon we should add `host_required` and `user_required` to `stricturl` to customize the full `AnyUrl`

Reply:
`host_required` is already present in `stricturl`, I just forgot to update it on the docs (will do).

In my view, adding the `user_required` field to `stricturl` (and docs) is outside the scope of this issue, perhaps it's better we split it into a new one? If you consider the change trivial enough to include it here (it's ~2 lines after all), I don't mind adding it.

Reply:
Added the` host_required` in `stricturl` docs, resolved conflicts (again).

Is there anything else pending for this PR?